perfkitbenchmarker/linux_packages/silo.py

```diff
@@ -33,6 +33,9 @@ def _Install(vm):
   vm.RemoteCommand('git clone {0} {1}'.format(GIT_REPO, SILO_DIR))
   vm.RemoteCommand('cd {0} && git checkout {1}'.format(SILO_DIR,
                                                        GIT_TAG))
+  # This is due to a failing clone command when executing behind a proxy.
+  # Replacing the protocol to https instead of git fixes the issue.
+  vm.RemoteCommand('git config --global url."https://".insteadOf git://')
```

Comment:
I switched to use https:// instead of git:// protocol - the latter one uses port 9418 which is blocked. Both http (80) and https (443) works fine. I'm quite confused with port 8080 and SSH - I don't mess up with these at all :)